packages/sync/src/links/LinksBuilder.ts

```diff
@@ -17,27 +33,43 @@ import { LocalDirectiveFilterLink } from "./LocalDirectiveFilterLink";
  * - Error handling
  * - Audit logging
  */
-export const defaultHttpLinks = async (config: DataSyncConfig): Promise<ApolloLink[]> => {
-  const offlineQueueLink = new OfflineQueueLink(config, "mutation");
-  const localDirectiveFilterLink = new LocalDirectiveFilterLink();
+export const defaultHttpLinks = async (config: DataSyncConfig): Promise<ApolloLink> => {
+  let links: ApolloLink[] = [];
+  if (config.networkStatus) {
```

Comment:
IMHO this will be always non null. Is that because of typescript bragging about this being undefined.

Reply:
yes, this was because of typescript warning

Reply:
Yeah there are two patterns to sort this:
- Create separate interface with no optionals (too much IMHO)
- use `config.networkStatus as NetworkStatus` to trick typescript.
- if like here. 

Let's leave it this way.